<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>func</class>
 <widget class="QWidget" name="func">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>475</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>出入人员名单</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>421</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>功能操作</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>401</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_face_select">
     <attribute name="title">
      <string>人脸搜索</string>
     </attribute>
     <widget class="QLabel" name="label_photo_select">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>281</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string>捕获图片</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_select">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Symbola</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>67</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>类型</string>
      </property>
     </widget>
     <widget class="QComboBox" name="group_face_select">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>240</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>学生</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>老师</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_face_register">
     <attribute name="title">
      <string>人脸注册</string>
     </attribute>
     <widget class="QLabel" name="label_photo_resgister">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>281</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string>捕获图片</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>类型</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>用户ID</string>
      </property>
     </widget>
     <widget class="QComboBox" name="group_face_register">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>学生</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>老师</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="ipt_face_register">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_face_register">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>220</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Symbola</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_face_update">
     <attribute name="title">
      <string>人脸更新</string>
     </attribute>
     <widget class="QLabel" name="label_photo_update">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string>捕获图片</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>用户ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>类型</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ipt_face_update">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="group_face_update">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>学生</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>老师</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btn_face_update">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>220</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Symbola</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>更新</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>用户信息：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="ipt_user_info">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>81</width>
        <height>131</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>人脸列表</string>
     </attribute>
     <widget class="QLineEdit" name="ipt_face_list">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>类型</string>
      </property>
     </widget>
     <widget class="QComboBox" name="group_face_list">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>学生</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>老师</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>用户ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_face_list">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Symbola</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>获取</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_user_select">
     <attribute name="title">
      <string>获取用户列表</string>
     </attribute>
     <widget class="QPushButton" name="btn_user_select">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Symbola</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>获取</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>类型</string>
      </property>
     </widget>
     <widget class="QComboBox" name="group_user_select">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>学生</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>老师</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>获取用户信息</string>
     </attribute>
     <widget class="QPushButton" name="btn_user_search">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Symbola</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>更新</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ipt_user_search">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>用户ID</string>
      </property>
     </widget>
     <widget class="QComboBox" name="group_user_search">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>学生</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>老师</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>类型</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_user_delete">
     <attribute name="title">
      <string>删除用户</string>
     </attribute>
     <widget class="QLineEdit" name="ipt_user_delete">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_user_delete">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Symbola</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>删除用户</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>用户ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>类型</string>
      </property>
     </widget>
     <widget class="QComboBox" name="group_user_delete">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>学生</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>老师</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>20</width>
     <height>411</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>361</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="group">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>300</height>
       </size>
      </property>
      <property name="title">
       <string>视频预览</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QVideoWidget" name="video" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_open">
        <property name="text">
         <string>打开摄像头</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="text">
         <string>关闭摄像头</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_get">
        <property name="text">
         <string>采集图像</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>50</y>
     <width>20</width>
     <height>411</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="result_box">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>60</y>
     <width>351</width>
     <height>371</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
